### PR TITLE
Dylan feature 2

### PR DESCRIPTION
**Pull request by dylan-smith**
explaining the feature

#JIRA-123

### CODE REVIEW

**Review by akonduri-mck — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "tsc",
     "watch": "tsc --watch",
-    "package": "ncc build src/main.ts -o dist",
+    "package": "ncc build src/main.ts",
```
Comment (dylan-smith) [inline package.json]: don't we need the output arg

File: src/replace.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ export async function replaceTokens(
     `${escapeDelimiter(tokenPrefix)}(.+?)${escapeDelimiter(tokenSuffix)}`
   );
 
-  const result = await replace({
+  const result = await replaceAll({
```
Comment (dylan-smith) [inline src/replace.ts:16]: this is a stupid name

File: tsconfig.json [hunk 3]
```diff
@@ -1,13 +1,13 @@
 {
   "compilerOptions": {
-    "target": "es6",
+    "target": "es5",
     "module": "commonjs",
     "outDir": "./lib",
     "rootDir": "./src",
     "strict": true,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```
Comment (dylan-smith) [inline tsconfig.json]:
```suggestion
    "noImplicitAny": true,
```

this should stay as true

File: package.json (hunk 1 above)
Comment (dylan-smith) [reply package.json]: oh maybe you're right, I'll add it back in

**Review by akonduri-mck — COMMENTED** [review]
too many changes too fast Dylan!!! :)

**Review by eqpk3ia — APPROVED** [review]

**Pull request merged**